tests/mock_network/mod.rs

```diff
@@ -23,7 +23,7 @@ use routing::{
 };
 use std::collections::BTreeMap;
 
-pub const LOWERED_ELDER_SIZE: usize = 3;
+pub const LOWERED_ELDER_SIZE: usize = 4;
```

Comment:
Why is it 4 now instead of 3?

Reply:
https://github.com/maidsafe/routing/pull/2077#discussion_r402318285